jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -176,14 +213,15 @@ public Entry reserve(int allotment)
      * @param idx the index of the entry to acquire.
      * @return the specified entry or null if there is none at the specified index or if it is not available.
      */
+    @Deprecated
```

Comment:
If we go with this design, the RR connection pool should be rewritten to use the RR strategy and this method should be removed.